src/index.js

```diff
@@ -48,6 +48,10 @@ export const queryRenderer = (rootQuery, variables) =>
           variables={vars}
           render={({ error, props, retry }) => {
             if (!props && !error) {
+              const toRenderDuringLoading = LoadingComponent
```

Comment:
This won't work as this variable isn't being used anywhere. Null is still being returned, are you sure about this implementation?

Reply:
My apologies, it seems that I had not pushed all the changes. I just fixed it!

Reply:
I pushed some new changes so that the correct value is returned!

Reply:
@ErrorPro Any updates? Thank you!